### PR TITLE
feat: Rename instances of rayiov1alpha1 to rayv1alpha1

### LINKED ISSUE #1079: [Refactor] Rename rayiov1alpha1 and rayv1alpha1 to v1alpha1
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/kuberay/issues) and found no similar issues.

### KubeRay Component

ray-operator

### 1. "kuberay/ray-operator/apis/ray/v1alpha1" 

In the KubeRay codebase, there are three aliases for v1alpha1: rayiov1alpha1, rayv1alpha1, and v1alpha1. Let's ensure consistency across all of them.

* Case 1: https://github.com/ray-project/kuberay/blob/b0649c43bbae3a0ac4917f94d80cbde466bb4f88/ray-operator/controllers/ray/utils/util.go#L21

* Case 2: https://github.com/ray-project/kuberay/blob/b0649c43bbae3a0ac4917f94d80cbde466bb4f88/ray-operator/controllers/ray/utils/dashboard_httpclient.go#L21

* Case 3: https://github.com/ray-project/kuberay/blob/b0649c43bbae3a0ac4917f94d80cbde466bb4f88/ray-operator/controllers/ray/common/rbac.go#L4

<del>Use Case 3 to replace Case 1 and Case 2.</del>

[Update]: Use Case 2 to replace Case 1 and Case 3.

### Reproduction script

No

### Anything else

_No response_

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by anishasthana**
cc @kevin85421  the PR is ready for review. I deployed using the instructions in development.md. I tested with the smallest sample: https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-cluster.mini.yaml

## Related issue number

Closed #1079

## Checks

- [x] I've made sure the tests are passing. 
- Testing Strategy
   - [x] Unit tests
   - [x] Manual tests

### CODE REVIEW

**Comment by anishasthana** [comment]
Closes #1079

File: ray-operator/controllers/ray/common/pod_test.go [hunk 1]
```diff
@@ -15,21 +15,21 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 
-	rayiov1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	ray747213v1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
```
Comment (kevin85421) [inline ray-operator/controllers/ray/common/pod_test.go:18]: What's `ray747213v1alpha1`?

Comment (anishasthana) [reply ray-operator/controllers/ray/common/pod_test.go]: Good catch. Looks like accidentally hit my yubikey mid edit. Fixed!

**Review by kevin85421 — APPROVED** [review]
LGTM!

**Pull request merged**